src/main/java/Task.java

```diff
@@ -0,0 +1,26 @@
+public class Task {
+    protected String taskName;
```

Comment:
```suggestion
    protected final String taskName;
```
Since `taskName` is only updated once (i.e. in your constructor), I feel it is good practice to add the `final` keyword. But, I don't think this is part of the coding standards so feel free to close this comment